acceptance/config/beaker/options.rb

```diff
@@ -8,6 +8,7 @@
   "puppetservice" => 'puppetserver',
   "use-service" => true,
   "master-start-curl-retries" => 60,
-  "puppetpath" => "/etc/puppetlabs/agent"
+  "puppetpath" => "/etc/puppetlabs/puppet",
+  "puppetserver-confdir" => '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
Looks like https://github.com/puppetlabs/puppet/blob/d8f95998fc4e3358ed94ea61d3406b7fa0174e98/acceptance/config/aio/options.rb#L5 has the `puppetserver-confdir` change, although it has it like this:

~~~~
:'puppetserver-confdir' => '/etc/puppetlabs/puppetserver/conf.d',
~~~~

Not sure the syntax -- leading colon -- makes a difference?

Is the `puppetpath` still needed?  Wasn't sure since I didn't see it in the .../aio/options.rb file, but maybe that file's not the best one to reference for this?